src/js/gui.js

```diff
@@ -74,7 +74,7 @@ const GuiControl = function () {
 };
 
 function GUI_checkOperatingSystem() {
-    return navigator.userAgentData.platform;
+    return navigator?.userAgentData?.platform || 'Android';
```

Comment:
I like it 🙂

Reply:
Navigator will be available always, but it does not hurt 😄